main.go

```diff
@@ -263,7 +273,7 @@ func main() {
 	var malwareManager malwaremanager.MalwareManagerClient
 	if cfg.EnableMalwareDetection {
 		// create exporter
-		exporter := exporters.InitExporters(cfg.Exporters, clusterData.ClusterName, nodeName)
+		exporter := exporters.InitExporters(cfg.Exporters, clusterData.ClusterName, nodeName, cloudMetadata)
```

Comment:
Food for thought: What are we not adding to the data in a way that allows Rules to access it? In theory we could make rules that take it into account

Reply:
In theory maybe but I don't think there is something there that is relevant for rules.